tests/nuodb_basic_test.py

```diff
@@ -172,6 +172,23 @@ def test_many_significant_digits(self):
     def test_numeric_no_decimal(self):
         self._test_decimal_fixture(decimal.Decimal("1.000"), 5, 3)
 
+    # This test case produces at least three different defects, perhaps
+    # more under the covers.
+    @unittest.skip("produces data errors for e-notations")
+    def test_enotation_decimal_large(self):
+        numbers = (
+            # Yields:  AssertionError: '400000000.00' != '4E+8'
+            decimal.Decimal('4E+8'),
```

Comment:
I'm not sure this is a valid test. Or, more specifically, the part of `test_decimal_fixture` that asserts on string equality. Is there any contract that requires the database to respond using scientific notation when a value was inserted that way?